### PR TITLE
Removing close button from right panel in moderation view

### PR DESCRIPTION
**Pull request by Rafi993**
## What type of PR is this?

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Hides close button in moderator view

## Related Tickets & Documents

closes https://github.com/forem/forem/issues/10265

### Article view
article_view
### Mod view
mod_view

## Added tests?

- [x] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.forem.com
- [ ] readme
- [x] no documentation needed

## What gif best describes this PR or how it makes you feel?

Happy

### CODE REVIEW

File: app/javascript/actionsPanel/__tests__/initializeActionsPanelToggle.test.js [hunk 1]
```diff
@@ -21,13 +21,24 @@ describe('toggling the actions panel', () => {
 
     test('it should have a click listener that toggles the appropriate classes', () => {
       initializeActionsPanel(path);
+
+      const modContainer = document.getElementById('mod-container');
+      modContainer.contentWindow.document.write(
```
Comment (Rafi993) [inline app/javascript/actionsPanel/__tests__/initializeActionsPanelToggle.test.js:26]: This for mocking contents of the iframe

**Review by citizen428 — APPROVED** [review]

**Review by Zhao-Andy — APPROVED** [review]
LGTM!

**Comment by Zhao-Andy** [comment]
Admin merge because CI passed but GitHub isn't reading it properly.

@Rafi993 thanks so much for the fix and PR! :)

**Comment by Rafi993** [comment]
> Admin merged because CI passed but GitHub isn't reading it properly.
> 
> @Rafi993 thanks so much for the fix and PR! :)

Thank you @Zhao-Andy

**Pull request merged**